test/object-store/util/sync/baas_admin_api.cpp

```diff
@@ -561,6 +566,18 @@ class Baasaas {
         }
     }
 
+    std::string get_current_stable_image()
+    {
+        auto [images, coid] = do_request("images", app::HttpMethod::get);
+        auto all_branches = images["allBranches"].template get<std::vector<std::string>>();
+        if (all_branches.size() < 2) {
+            throw RuntimeError(
+                ErrorCodes::MalformedJson,
+                util::format("No stable branch found in baasaas response %1 (coid: %2)", images.dump(), coid));
+        }
+        return all_branches[1];
```

Comment:
Should we add a comment as to why we're grabbing the second branch in the list? (e.g. "skip over the `master` branch which is at index 0")